src/views/Viewer.vue

```diff
@@ -1003,6 +1085,11 @@ export default {
 			background-color: transparent;
 			box-shadow: none;
 		}
+		.modal-container-fullscreen {
+			top: 0;
+			bottom: 0;
+			height: 100%;
+		}
```

Comment:
I guess we would not need that if we set the modal element as full screen instead of the modal ?

Reply:
Even with the modal element made full screen this will be required - the CSS rule above sets `top` and `bottom` properties to `--header-height` (50px AFAIK), to leave space for the header and have symmetric space at the bottom.
https://github.com/nextcloud/viewer/blob/660d1fc5be91e2966194dce3d8491863ff3703fc/src/views/Viewer.vue#L1081

As we want to use 100% of available screen height, we must set top/bottom to 0.